src/services/Login.ts

```diff
@@ -0,0 +1,154 @@
+import * as API from './Api';
+
+import { AuthInfo, AuthStrategy, AuthResult } from '../types/Auth';
+import { LoginSession } from '../store/Store';
+import { TimeInMilliseconds } from '../types/Common';
+
+// We just mock the types for state for now, to avoid injecting more
+// dependencies on the service.
+type Dispatch = any;
+type AppState = any;
```

Comment:
Isn't `AppState` supposed to be `KialiAppState`. If so, why the downgrade to `any`?